tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -19,6 +19,11 @@ def __init__(self, **kwargs):
                          default=False,
                          help='restart target instance between tests')
 
+        group.add_option('--locale',
+                         default="undefined",
```

Comment:
Yes, the default language of the phone immediately after the flash of a new build is en-US, but after that the phone boots up with whatever language that was previously set. IF we put a default language value, it might be quite a change to non-english testers.